variables.tf

```diff
@@ -146,3 +146,20 @@ variable "resources" {
   }
   description = "Allows to set cpu/memory resources Limits/Requests for deployment/cronjob"
 }
+
+variable "secret_id" {
+  description = "Secret ID"
+  type = string
+}
+
+variable "secret_description" {
```

Comment:
why we need `secret_description` and `recovery_window_in_days` variables?
can we have `secret_id` optional(with default value set), and also some additional mapping of secret keys to our variables?

Reply:
secret_description is an optional field

Reply:
If you don't assign recovery_window_days, then by default it will be 30 days

Reply:
Yes, we can assign default secret_id

Reply:
I can't see the places where recovery_window_in_days and secret_description variables are used, can you show where in this module codes those variables you use?

can you set default value for `secret_id`, so that it will not be required. Using/pulling credentials we need from secret should be optional/configurable so this secret_id can have empty value by default

Reply:
I added an example of using Secrets Manager module, please review again